sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -82,7 +82,7 @@ int main()
 
   az_log_classification const classifications[] = { AZ_LOG_HTTP_RESPONSE, AZ_LOG_END_OF_LIST };
   az_log_set_classifications(classifications);
-  az_log_set_callback(test_log_func);
```

Comment:
Fyi, @danieljurek, looks like this samples change didn't cause a CI build failure that I expected.

Reply:
How does one build the samples? Is this done through the normal `cmake ..` and `cmake --build .` done from a `build` folder at the root of the repo or is there some additional target that needs to be specified?

Reply:
@vhvb1989 can you provide some insight here on how we build the samples? I was looking for some opion like `BUILD_SAMPLES` within the docs, but couldn't find one.

Looking at yml, I am guessing it has to do with these options:
` -DTRANSPORT_CURL=ON -DTRANSPORT_PAHO=ON -DAZ_PLATFORM_IMPL=WIN32`
https://github.com/Azure/azure-sdk-for-c/blob/43e4de8dcac97b144c24e0693028c88a17b98b91/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L41-L56

Reply:
keyvault is dead on C.
I though @RickWinter  had it removed before.  Probably we just removed the code but the samples stayed there. 
However, this is zombie code 🤣 We can remove the entire folder `sdk/samples/keyvault`

About the question,  yes, @ahsonkhan  you have it right. Those flags are the right ones